os/net/lwip/src/api/sockets.c

```diff
@@ -1498,7 +1529,7 @@ int lwip_select(int maxfdp1, fd_set *readset, fd_set *writeset, fd_set *exceptse
 			if ((readset && FD_ISSET(i, readset)) || (writeset && FD_ISSET(i, writeset)) || (exceptset && FD_ISSET(i, exceptset))) {
 				struct lwip_sock *sock;
 				SYS_ARCH_PROTECT(lev);
-				sock = tryget_socket(i);
+				sock = tryget_socket(i, pid);
```

Comment:
I can't find a local variable for pid. Is that a global variable?

Reply:
thanks and corrected now.